src/web/mjs/connectors/Sukima.mjs

```diff
@@ -0,0 +1,191 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Sukima extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'sukima';
+        super.label = 'Sukima';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://www.sukima.me';
+        this.language = 'japanese';
```

Comment:
May remove this unused field

Reply:
I dream of a feature that reads the language from `this.language` instead of `this.tags`. But I guess it is silly to do it now since a lot of other connectors need to be amended as well.